quay-enterprise/bitbucket-app.md

```diff
@@ -0,0 +1,19 @@
+# Creating an OAuth Application in BitBucket
+
```

Comment:
Can you add a quick description of what you're doing here? Why do I care if I stumbled upon this page?